cluster-scope/base/core/namespaces/ai4cloudops/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yaml
+- ../../../../components/project-admin-rolebindings/ai4cloudops
```

Comment:
I'm not sure about this: it's the *only* resource in `cluster-scope/base` that includes something from a parent directory. Do we really need a one-off here? We're not including these resources in any cluster overlays right now, but if this is intended for the production cluster we should be handling this via coldfront instead of by manually creating the namespace.